executions/graphql-kotlin-dataloader/src/main/kotlin/com/expediagroup/graphql/dataloader/KotlinDataLoader.kt

```diff
@@ -24,5 +25,7 @@ import org.dataloader.DataLoader
  */
 interface KotlinDataLoader<K, V> {
     val dataLoaderName: String
-    fun getDataLoader(): DataLoader<K, V>
+    fun getDataLoader(graphQLContext: GraphQLContext?, graphQLContextMap: Map<*, Any>?): DataLoader<K, V> = getDataLoader()
```

Comment:
If we're ok with only supporting the context map, things get a little cleaner?  if so, the build.gradle.kts changes to include graphql-java instead of the schema generator

Reply:
+1 to just context map as otherwise it is adding deprecated functionality